packages/orbit-components/src/common/consts.ts

```diff
@@ -1,21 +1,3 @@
-export enum PLACEMENTS {
```

Comment:
What's the reason for moving this to a different file and separating the `auto` ones from the others?

Reply:
This was moved to a different file because there were more types and enums to export and so I thought it would make sense to keep them in a single dedicated file.

The separation from `auto` happened because popper's type `Placement` used to include the `auto`. But that is not the case with `floating-ui`, so I needed some workarounds to keep supporting it.
Since the enums map the types, and also because of the `isFixedPlacement`, I thought it would be safer to keep them separated